blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/QueueContainerImpl.java

```diff
@@ -33,9 +38,39 @@ public BlueQueueItem get(String name) {
                 return blueQueueItem;
             }
         }
+        // JENKINS-38540 - To make this consistent with the activity API, check the runs, too
+        String runId = findQueueInRuns(name);
+        if (runId != null) {
+            try {
+                StaplerResponse rsp = Stapler.getCurrentResponse();
+                // Send a redirect, not sure the specific code which would be best. substring to fix double slash
+                rsp.sendRedirect(Jenkins.getInstance().getRootUrl() + pipeline.getLink().toString().substring(1) + "runs/" + runId);
```

Comment:
This works as expected although the redirect URL doesn't end with a forward slash. Whenever the browser requests a URL without one, Stapler does a redirect to a URL with the forward slash added. So the end result is a little funky in the network debugger:

/queue/5/ -> /runs/25 -> /runs/25/

Could you add a util function that ensures we always get a trailing forward slash on these URL's? It would be easier to debug and should also be more efficient.